syntax = "proto3";
package message;

message GetNameCMD {
    uint64 protocol_id = 1;
    uint64 cmd_id = 2;
}